lib/src/cli/flutter_cli.dart

```diff
@@ -379,8 +379,8 @@ Future<int> _flutterTest({
       }
     },
     onError: (Object error, StackTrace stackTrace) {
-      subscription.cancel();
-      completer.completeError(error, stackTrace);
+      stderr('$clearLine$error');
```

Comment:
The test process throws errors for warnings and it turns out we actually don't want to treat those as errors and instead just propagate them to stderr for the user to see without terminating the process.